plugins/snapshots/windows/cimfs.go

```diff
@@ -78,9 +92,21 @@ func NewCimFSSnapshotter(root string) (snapshots.Snapshotter, error) {
 		return nil, err
 	}
 
+	// create base scratch VHD that is used for every scratch snapshot, if it doesn't already exist
+	baseVhdPath := filepath.Join(baseSn.root, baseVHDName)
```

Comment:
Can a single VHD used for unpacking multiple images simultaneously pose any concurrency/data consistency problems?

Reply:
We don't write anything to the VHD when importing images. And each container gets its own copy of the scratch VHD so this shouldn't be a problem.